traffic_ops/traffic_ops_golang/config.go

```diff
@@ -123,6 +126,17 @@ func LoadConfig(cdnConfPath string, dbConfPath string) (Config, error) {
 		return Config{}, fmt.Errorf("unmarshalling '%s': %v", dbConfPath, err)
 	}
 	cfg, err = ParseConfig(cfg)
+
+	riakConfBytes, err := ioutil.ReadFile(riakConfPath)
+	if err != nil {
+		return cfg, fmt.Errorf("reading riak conf '%v': %v", riakConfPath, err)
+	}
+	riakconf, err := getRiakAuthOptions((string)(riakConfBytes))
```

Comment:
parens around (string) are not needed -- `string(riakConfBytes)`

Reply:
Oops, Hard to shake C sometimes, looks like a cast doesn't it :-) . I'll fix it.